### DISCUSSION TITLE
2019 Modular Avatar unitypackage

### DISCUSSION THREAD

**Discussion by BIGM8** [bdunderscore/modular-avatar#1117] [General]
Where can I find the 2019 Modular Avatar Unity package to download?

**Comment by anatawa12** [comment]
Due to technical reasons, unitypackage are no longer be released since 1.8.0. We have to use VPM client like to install newer modular avatar. If you're using Windows, you can use VCC, the official GUI VPM Client. For other OSes, you may use VPM CLI (the official CLI VPM Client), vrc-get (third-party CLI VPM Client), or ALCOM (third-party crossplatform GUI VPM client).

Since Modular Avatar 1.10.0, Unity 2019 will no longer be supported. Therefore, you have to use 1.9.x for 2019 projects. (VPM Client will check Unity Version and should install latest compatible version)

**Reply by BIGM8** [reply]
So how do I import it into my 2019 project?

**Reply by anatawa12** [reply]
There's no difference for installation method between 2019 project and 2022 project.

"Download (using VCC)" button in https://modular-avatar.nadena.dev/ would open dialog shows how to install modular-avatar with VCC.
